.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -25,55 +25,55 @@ parameters:
 - name: buildAzureLinuxV2gen1
   displayName: Build AzureLinuxV2 Gen1
   type: boolean
-  default: false
+  default: true
```

Comment:
lets remember to update sig config starting with the next release

Reply:
starting with this release itself, I am going to rebuild VHDs